Applications/relax2/server/relax_server_dev.c

```diff
@@ -8443,7 +8443,9 @@ int main(int argc)
   tx_size = ((uint16_t *)(cfg + 12));
 
   //rx_switch
-  rx_switch = ((uint16_t *)(cfg + 6));
+  // the offset is a bit obscure here, but cfg1 on the FPGA falls on the same PAGE as cfg as far as the OS is concerned,
+  // so I'm using the same mmap here
```

Comment:
Is this comment still valid?

Reply:
Ii isn't, will cleanup in the follow-on cleanup PR.